controllers/attractor/lb-fe.go

```diff
@@ -155,7 +149,6 @@ func (l *LoadBalancer) insertParameters(dep *appsv1.Deployment) *appsv1.Deployme
 
 	ret.Spec.Template.ObjectMeta.Labels["app"] = lbFeDeploymentName
 	ret.Spec.Template.Spec.Affinity.PodAntiAffinity.RequiredDuringSchedulingIgnoredDuringExecution[0].LabelSelector.MatchExpressions[0].Values[0] = lbFeDeploymentName
-	ret.Spec.Template.Spec.ServiceAccountName = common.ServiceAccountName(l.trench)
```

Comment:
Does it mean lb-fe doesn't need any roles?

Reply:
right, the service account gives the access to the kubernetes API (watch configmap), and now only the NSP pod requires it